apps/devhub/views.py

```diff
@@ -1735,3 +1741,18 @@ def docs(request, doc_name=None):
 def search(request):
     query = request.GET.get('q', '')
     return render(request, 'devhub/devhub_search.html', {'query': query})
+
+
+def api_key_agreement(request):
```

Comment:
This looks exactly the same as [submit()](https://github.com/Nolski/olympia/blob/signing-api-agreement/apps/devhub/views.py#L1389) except that it redirects to `devhub.api_key` afterwards. Instead, you could make `submit()` more reusable like this:
````
def submit(request, step, next_step=_step_url(2)):
    ...
    return redirect(next_step)
````
Then in this view:
````
def api_key_agreement(request):
    return submit(request, next_step=reverse('devhub.api_key'))
````